stubs/core.stub

```diff
@@ -2,7 +2,15 @@
 
 /**
  * @param ($allow_string is false ? object : object|string) $object_or_class
- * @param class-string $class
```

Comment:
changed both, `is_a` and `is_subclass_of `$class` arg back to `string` for max compat with 1.6.x